setup.py

```diff
@@ -43,6 +43,7 @@ def read_file(filename, encoding='utf8'):
     'pytz',
     'requests',
     'setuptools',  # pkg_resources
+    'six',
```

Comment:
Since the `letsencrypt` module isn't py3+ compatible and we aren't currently working towards that goal, I don't think we should add `six` as a dependency.